source/backend/api/Services/LeaseService.cs

```diff
@@ -471,5 +489,67 @@ private void AppendNewItemsToChecklist(PimsLease lease, ref List<PimsLeaseCheckl
                 }
             }
         }
+
+        private void ValidateRenewalDates(PimsLease lease, ICollection<PimsLeaseRenewal> renewals)
+        {
+            if (lease.LeaseStatusTypeCode != PimsLeaseStatusTypes.ACTIVE)
```

Comment:
Is this accurate? it seems like dates are optional when a lease is not active, but should the user be allowed to enter overlapping dates when not active?

Reply:
Thats correct. Check reply above.